src/bun.js/javascript.zig

```diff
@@ -626,6 +626,12 @@ pub const VirtualMachine = struct {
             bun.reloadProcess(bun.default_allocator, !strings.eqlComptime(this.bundler.env.map.get("BUN_CONFIG_NO_CLEAR_TERMINAL_ON_RELOAD") orelse "0", "true"));
         }
 
+        if (!strings.eqlComptime(this.bundler.env.map.get("BUN_CONFIG_NO_CLEAR_TERMINAL_ON_RELOAD") orelse "0", "true")) {
+            Output.flush();
+            Output.disableBuffering();
+            Output.resetTerminalAll();
```

Comment:
we should re-enable buffering after this line

otherwise LGTM

Reply:
Should this also be done at https://github.com/oven-sh/bun/blob/main/src/bun.zig#L1311?

Reply:
thank you for the explanation 🙂

Reply:
> Will do, should this also be done at https://github.com/oven-sh/bun/blob/main/src/bun.zig#L1311?

For anyone else reading from the future, the permalink for the above was https://github.com/oven-sh/bun/blob/76dee9312ba5201583ba9f8232b96397fa490808/src/bun.zig#L1311